src/dune_console/dune_console.ml

```diff
@@ -131,26 +143,66 @@ module Backend = struct
 
   let spawn_thread = Fdecl.create Dyn.opaque
 
-  let threaded (module Base : S) : (module S) =
+  (** Backends may have access to an internal state. This type is used by the
+      [threaded] backends. *)
```

Comment:
```suggestion
      [Threaded] backends. *)
```

Reply:
That is, it seems to me that the `Threaded` module type is what defines what it means for a backend to be threaded, not the `threaded` function.

Reply:
In my mind, `threaded` creates the threaded backends and `Threaded` is just the data to do so, not necessarily the data of a threaded backned.

Reply:
But I can change it if you think it makes more sense.